hockeysdk/src/main/AndroidManifest.xml

```diff
@@ -6,6 +6,7 @@
     <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
     <uses-permission android:name="android.permission.INTERNET" />
     <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE" />
+    <uses-permission android:name="android.permission.REQUEST_INSTALL_PACKAGES" />
```

Comment:
Just an FYI for @ranterle – we're bumping minAPI to 15, and in that, trying to get rid of WRITE_EXTERNAL_STORAGE.  The new permission is required on apiLevel 25+